doc/getting_started/installation_linux.rst

```diff
@@ -160,6 +184,17 @@ Follow these steps to install the SDK on your Linux host system.
      export ZEPHYR_SDK_INSTALL_DIR=/opt/zephyr-sdk
      EOF
 
+  .. note::
```

Comment:
Same comment here, maybe change the note to be:

    Some Linux distributions have default CFLAGS and CXXFLAGS
    environment variables already set.  For all distros, they need to be
    unset to prevent these settings from interfering with cmake: